src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -341,6 +342,7 @@ def import_mailboxes(self, domain):
                             # secondary email is mandatory. Unfortunately, dimail doesn't
                             # store any. We temporarily give current email as secondary email.
                             status=enums.MailboxStatusChoices.ENABLED,
+                            password=make_password(None),
```

Comment:
This could use a comment. Are we absolutely sure it can't lead to successful with empty passwords?

Reply:
Yes, when `None` it starts with the "invalid password" prefix.

See https://github.com/django/django/blob/5.1.6/django/contrib/auth/hashers.py#L103-L106
and https://github.com/django/django/blob/5.1.6/django/contrib/auth/hashers.py#L28-L42

I will add a comment. I don't like testing the framework but I can test "None" password cannot login, just in case.